src/app.ts

```diff
@@ -166,7 +166,7 @@ export function newApp(
   );
 
   const obfuscateToken = (originalUrl: string) =>
-    originalUrl.replace(/([?&]token=)[^&]*(&?.*)/, "$1REDACTED$2");
+    originalUrl.replace(/([?&]token=|access_token=)([^&]*)/g, "$1REDACTED");
```

Comment:
shouldn't this be:

`([?&]token=|[?&]access_token=)([^&]*)/g`

?

Reply:
or we can just use `/(token=|access_token=)([^&]*)/g` ?

Reply:
@gunzip you are right. The correct regex is `([?&]token=|[?&]access_token=)([^&]*)/g`.

Instead `/(token=|access_token=)([^&]*)/g` can't handle correctly the exact parameter name, es. `px_token=123` will be offuscated [example here](https://regex101.com/r/tqBIf1/1)

Reply:
Fixed in 18b2684